superset/cli.py

```diff
@@ -46,6 +46,9 @@ def runserver(debug, address, port, timeout, workers):
             threaded=True,
             debug=True)
     else:
+        print("DEPRECATED: Please run gunicorn via the 'runserver_gunicorn' command")
```

Comment:
Nothing should change from the user perspective when switching to the new command. I'm keeping the old one in there for now for backwards compatibility. At least inside Airbnb, it's hard for us to simultaneously switch both the app code and the command used to run the app (set via chef). 

Once this change is in, I can come back in a few days and delete the old gunicorn command.

Let me know if this doesn't sound ok.